src/modules/10-common/pages/welcome/SelectModuleList.tsx

```diff
@@ -42,6 +59,60 @@ const SelectModuleList: React.FC<SelectModuleListProps> = ({ onModuleClick, modu
   const { mutate: updateDefaultExperience, loading: updatingDefaultExperience } = useUpdateAccountDefaultExperienceNG({
     accountIdentifier: accountId
   })
+  const { mutate: createProject, loading: creatingProject } = usePostProject({
+    queryParams: {
+      accountIdentifier: '',
+      orgIdentifier: ''
+    }
+  })
+  const [isFetchingOrg, setIsFetchingOrg] = useState<boolean>(false)
+  const { mutate: createOrg, loading: creatingOrg } = usePostOrganization({
+    queryParams: {
+      accountIdentifier: accountId
+    }
+  })
+
+  const createDefaultProjectAndRedirect = React.useCallback((orgId: string) => {
+    createProject(
+      {
+        project: {
+          identifier: DEFAULT_PROJECT_ID,
+          name: DEFAULT_PROJECT_NAME,
+          orgIdentifier: orgId
+        }
+      },
+      {
+        queryParams: {
+          accountIdentifier: accountId,
+          orgIdentifier: orgId
+        }
+      }
+    ).then((createProjectResponse: ResponseProjectResponse) => {
+      const { data: projectData, status: createProjectResponseStatus } = createProjectResponse
+      if (createProjectResponseStatus === 'SUCCESS' && projectData?.project.identifier) {
+        history.push(
+          routes.toCIGetStarted({
+            accountId,
+            module: 'ci',
+            orgIdentifier: orgId,
+            projectIdentifier: projectData.project.identifier
+          })
+        )
+      }
+    })
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook React.useCallback has missing dependencies: 'accountId', 'createProject', and 'history'. Either include them or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9254902)
```suggestion
  }, [accountId, createProject, history])
```